timemachine/fe/atom_mapping.py

```diff
@@ -137,13 +136,16 @@ def get_cores(
     max_visits: int
         Maximum number of nodes we can visit for a given threshold.
 
-    connected_core: bool
-        Set to True to only keep the largest connected
-        subgraph in the mapping. The definition of connected
+    max_connected_components: int or None
+        Set to k to only keep the largest k connected
+        subgraphs in the mapping. The definition of connected
         here is different from McGregor. Here it means there
         is a way to reach the mapped atom without traversing
         over a non-mapped atom.
 
+    min_connected_component_size: int
+        Minimum size of a connected component to be considered a valid mapping
+
```

Comment:
Not a native english speaker, but is it obvious that `max_connected_components` is leq `<=` (as opposed to `<`) and `min_connected_component_size` is geq `>=` (as opposed to `>`)?

Reply:
I would generally interpret max/min parameters as inclusive, but can clarify in the docstring

Reply:
Improved comments in https://github.com/proteneer/timemachine/pull/1349/commits/9f0bac8cc167008dffa7fdad2fc0e12909a7d4cc

Reply:
Clarified in https://github.com/proteneer/timemachine/pull/1349/commits/804729bcfed73a8e05d032c661287232d6b75ec6